### PR TITLE
Document process to update Fortran dependencies

### PR DESCRIPTION
**Pull request by tukss**
This explains how to use `make fdeps` and the parallel make. Please comment if it is understandable.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,8 @@ make.inc
 /src/build/
 /src/build/compile
 
+/doc/build
```
Comment (gassmoeller) [inline .gitignore]: Good idea.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -75,6 +75,12 @@ changes to the code at once). Please do
 not hesitate to ask questions about the workflow on the mailing list if you are
 not sure what to do.
 
+If you add new Fortran files or change the module structure of Rayleigh, the
+dependencies in the makefile. This is done by running `make fdeps` from the
```
Comment (gassmoeller) [inline CONTRIBUTING.md]:
```suggestion
dependencies in the makefile have to be updated. This is done by running `make fdeps` from the
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -75,6 +75,12 @@ changes to the code at once). Please do
...
+main repository directory, which modifies the file `src/Makefile.fdeps`. Commit
+this file along with your changes. You need the `makedepf90` tool on your
+development machine to perform this update.
```
Comment (gassmoeller) [inline CONTRIBUTING.md]:
```suggestion
development machine to perform this update. `makedepf90` is available in most package managers.
```

File: CONTRIBUTING.md (hunk 2 above)
Comment (tukss) [reply CONTRIBUTING.md]: Thanks. I never notice when I stop mid-sentence.

File: CONTRIBUTING.md (hunk 3 above)
Comment (tukss) [reply CONTRIBUTING.md]: I wish that was true. :wink: But at least Debian and Conda have it.

**Pull request merged**